pkg/clusteraccess/labels.go

```diff
@@ -0,0 +1,55 @@
+package clusteraccess
+
+import (
+	"slices"
+	"strings"
+)
+
+// L is a constructor for Label.
+func L(key, value string) Label {
```

Comment:
Can we maybe move this out of the `clusteraccess` package since we might want to use it in other places like resource mutators and maybe use something similar for annotations?